test/syscall_test/blocklists.exfat/chmod_test

```diff
@@ -4,4 +4,8 @@ ChmodTest.FchmodFileSucceeds_NoRandomSave
 ChmodTest.FchmodatFileAbsolutePath
 ChmodTest.FchmodatFile
 ChmodTest.ChmodFileToNoPermissionsSucceeds
-ChmodTest.FchmodFileToNoPermissionsSucceeds_NoRandomSave
\ No newline at end of file
+ChmodTest.FchmodFileToNoPermissionsSucceeds_NoRandomSave
+ChmodTest.FchmodFileSucceeds
+ChmodTest.FchmodDirSucceeds
+ChmodTest.FchmodatWithOpath
+ChmodTest.FchmodFileToNoPermissionsSucceeds
```

Comment:
Why have so many test cases been added to the blocklist? Are these cases failing on RISC-V, or are they now failing on x86?

Reply:
I mentioned it in the PR description. Nix-built initrd uses the latest and upstream gVisor (ver. 20240527.0). And there are lots of newly added tests, and those that may fail are added to blocklists.

They failed on x86. And I don't think gVisor's syscall tests support RISC-V.